AutomatedTest/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.5.3</version>
+        <version>2.7.3</version>

```

Comment:
Please refer to root level springboot starter so that all its version can be maintained through main pom file